src/nav/apm.yml

```diff
@@ -1,70 +1,13 @@
 title: APM agents
 path: /docs/apm
 pages:
-  - title: Getting started
+  - title: Get started with APM
```

Comment:
Not too sure about this title

Reply:
I'll leave some suggestions on CodeStream.